CMakeLists.txt

```diff
@@ -11,10 +11,9 @@ set(CMAKE_AUTORCC ON)
 set(CMAKE_CXX_STANDARD 17)
 set(CMAKE_CXX_STANDARD_REQUIRED ON)
 
-find_package(Qt5 REQUIRED COMPONENTS Core Widgets Charts SerialPort Bluetooth)
+find_package(Qt5 REQUIRED COMPONENTS Core Widgets SerialPort Bluetooth PrintSupport)# OpenGL)
```

Comment:
Do we need the module `PrintSupport` already?

Reply:
It says it [in the QCustomPlot docs](https://www.qcustomplot.com/index.php/tutorials/settingup) that we need it. Whether it still works without it, I haven't tried.

Reply:
Checked with the docu, this module is really needed. (Build fails without...)
[QCustomPlot Troubleshooting](https://www.qcustomplot.com/index.php/tutorials/settingup#:~:text=Linking%20aborts%20with%20%22Undefined%20reference%20to%20QPrinter(...)%22)